converter/src/main/java/gov/cms/qpp/conversion/model/Node.java

```diff
@@ -217,7 +217,7 @@ public void addChildNodes(Node... childNodes) {
 	 * @param childNode Node
 	 */
 	public void addChildNode(Node childNode) {
-		if (childNode == null || childNode == this) {
+		if (childNode == null || childNode == this) { // checking identity equals on purpose
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the "equals" method if value comparison was intended. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1698)